src/config/dynamic-theme-fixes.config

```diff
@@ -613,6 +613,13 @@ CSS
 
 ================================
 
+bab.la
+
+INVERT
+.logo-flash
```

Comment:
This element seems to be invisible for me. Any specific scenario where this is visible?

Reply:
Just scroll down and look at the top left corner